app/models/product_test.rb

```diff
@@ -202,31 +203,64 @@ def self.most_recent
 
   private
 
+  def gather_patient_ids
+    aggregate_id_list = []
+    # if it's a cvu_plus product
+    if product.cvuplus
+      # then check if vendor and/or patients are included in product
+      if product.vendor_patients
+        # If so, add appropriate vendor patient ids
+        aggregate_id_list.concat product.vendor.patients.pluck(:id)
```

Comment:
This should be adding vendor patients that are associated with the selected bundle- not all vendor patients. Will cause issues with a vendor that has patients across multiple bundles. Could re-add "def vendor_patient_ids" convenience method or similar code:
product.vendor.patients.where(bundleId: bundle.id).pluck(:id)